### PR TITLE
pdctl, api, schedule: pdctl supports scatter region.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: server/schedule/region_scatterer.go [hunk 1]
```diff
@@ -124,12 +125,13 @@ func (r *RegionScatterer) scatterRegion(region *core.RegionInfo) *Operator {
...
 		steps = append(steps, op.steps...)
 		steps = append(steps, TransferLeader{ToStore: newPeer.GetStoreId()})
+		kind |= op.Kind()
 	}
 
 	if len(steps) == 0 {
 		return nil
 	}
-	return NewOperator("scatter-region", region.GetId(), OpAdmin, steps...)
+	return NewOperator("scatter-region", region.GetId(), kind, steps...)
```
[the remark this answers is not in the document]
Comment (disksing) [reply server/schedule/region_scatterer.go]: Yep. To avoid the cluster being too busy.

[the remark this answers is not in the document]
Comment (huachaohuang) [reply server/schedule/region_scatterer.go]: But we don't want to limit it when using in TiDB Lightning, any way to satisfy both sides?

[the remark this answers is not in the document]
Comment (disksing) [reply server/schedule/region_scatterer.go]: Lightning will not be affected. With this patch, _scatter-region_ operators will occupy the limits of balance schedulers.

[the remark this answers is not in the document]
Comment (huachaohuang) [reply server/schedule/region_scatterer.go]: Oh, I see, the `ScatterRegion` API bypasses the rate limiter.

**Pull request merged**